app/controllers/api/stories_controller.rb

```diff
@@ -16,7 +16,7 @@ def resource
   end
 
   def resources
-    @stories ||=  resources_base.includes({audio_versions: [:audio_files]}, {account: [:image, :address, {opener:[:image]}]}, {series:[:image, :account]}, :images, :license)
+    @stories ||= resources_base.includes({audio_versions: [:audio_files]}, {account: [:image, :address, {opener:[:image]}]}, {series:[:image, :account]}, :images, :license)
```

Comment:
Line is too long. [172/80]<br>Space missing after colon.<br>Space inside { missing.<br>Space inside } missing.